bin/gtuber-dl/gtuber-dl.1

```diff
@@ -0,0 +1,80 @@
+.TH GTUBER-DL "1" "May 2023" "gtuber-dl 0.4.0" "User Commands"
+.SH NAME
+gtuber-dl \- manual page for gtuber-dl 0.4.0
+.SH DESCRIPTION
+.SS "Usage:"
+.IP
+gtuber\-dl [OPTION?] URI
```

Comment:
The three dots character seem incorrectly converted.

Reply:
Good catch! One has to run help2man with `--locale=C.UTF8` or wait for this bug to get fixed: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1036357